ideas/163-support-erc721.md

```diff
@@ -0,0 +1,78 @@
+## Preamble
+
+    Idea: 163
+    Title: Support ERC721 tokens
+    Status: Draft
+    Created: 2018-04-10
+    
+## Summary
+Add ERC721 support to Status wallet so that users can see their CryptoKitties, Punks, Celebrities, etc. alongside their other assets.
+
+## Swarm Participants
+- Lead/PM: [@rachelhamlin](https://github.com/rachelhamlin)
+- Testing & Evaluation: 
+- Contributor: [@jeluard](https://github.com/jeluard)
+- Contributor: [@goranjovic](https://github.com/goranjovic)
+- UX: [@denis-sharypin](https://github.com/denis-sharypin)
+
+## Product Overview
+
+With 100+ digital collectibles on the market and a combined sales volume greater than $50MM, it's important that we allow users to access their NFT collections inside Status right alongside their ETH and ERC20 tokens.
```

Comment:
To make this document more accessible, please define what `NFT` means before using the acronym.

Reply:
@martinklepsch ERC-721 token known as a non-fungible token (NFT)

Reply:
@martinklepsch sure thing.